inference_cli/server.py

```diff
@@ -58,6 +59,13 @@ def start(
             help="Roboflow API key (default is None).",
         ),
     ] = None,
+    tunnel: Annotated[
+        bool,
+        typer.Option(
+            "--tunnel",
+            help="Start a tunnel to expose inference to external requests"
```

Comment:
"Start a tunnel to expose inference to external requests on a TLS-enabled https://\<subdomain\>.roboflow.run" domain"